changelog/unreleased/auto-accept-shares.md

```diff
@@ -1,4 +1,4 @@
-Enhancement: Auto-Accept Shares
+Change: Auto-Accept Shares
 
 Automatically accept shares when configured by the user or admin
```

Comment:
shouldn't we also state that this is the default now and admins can opt out?

Reply:
Yes we should